study.md

```diff
@@ -131,17 +167,20 @@ Using Ruby's `select` Array method and JavaScript's `filter` method, create a
 new array consisting of the odd numbers of the existing array.
 
 Ruby:
-
+http://ruby-doc.org/core-2.3.1/Array.html#method-i-select-21
+<!--returns the odd numbers in the array-->
 ```ruby
 array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
-
-# your answer here
+array.select { |num| num.odd? }
 ```
 
 JavaScript:
-
+https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter
+<!--returns the odd numbers in the array-->
 ```javascript
 const array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
-
-// your answer here
+const numbers = array.filter(function(arrays) {
+  return arrays % 2
+})
+your answer here
 ```
```

Comment:
just be aware that 'arrays' in the way that you used it actually represents each individual element in 'const array'